### PR TITLE
Deblur==1.0.2

### PR DESCRIPTION
**Pull request by wasade**
Bump to deblur==1.0.2

### CODE REVIEW

**Comment by wasade** [comment]
I'm not sure how my changes are associated to the test failure...

**Comment by josenavas** [comment]
I think that is a result of a change in Qiita (not sure when it happened). Can you move [this line of .travis.yml](https://github.com/qiita-spots/qp-deblur/blob/master/.travis.yml#L38) to [here](https://github.com/qiita-spots/qp-deblur/blob/master/.travis.yml#L20)? That should make it.

**Comment by wasade** [comment]
cool cool, lets see if that works

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +26,8 @@ install:
   # installing this plugin
   - conda create --yes -n env_name python=$PYTHON_VERSION pip numpy scipy nose flake8 h5py
   - source activate env_name
-  - conda install --yes -c https://conda.anaconda.org/biocore VSEARCH MAFFT SortMeRNA
+  - conda install --yes -c bioconda VSEARCH==2.0.3 MAFFT=7.221 SortMeRNA==2.0
+    conda install --yes -c biocore deblur
```
Comment (josenavas) [inline .travis.yml]: I think you're missing the `-` character in here

**Pull request merged**